PeriodogramAnalysis/test_power_density_statistics.py

```diff
@@ -0,0 +1,96 @@
+from typing import Generator, Any
+
+import numpy as np
+import pytest
+from miv.core.datatype import Signal
+from power_density_statistics import (
+    SpectrumAnalysisBase,
+    SpectrumAnalysisPeriodogram,
+    SpectrumAnalysisWelch,
+)
+
+
+def mock_signal_generator():
```

Comment:
`pytest.fixture`

Reply:
Have added pytest.fixture to all three `test_` code.